py-nominal-streaming/src/point.rs

```diff
@@ -0,0 +1,273 @@
+//! Helpers for translating Python arguments into nominal_streaming types.
+//!
+//! Exposed surface (used by the pyo3 class):
+//!   - parse_timestamp(): int ns or datetime.datetime → google.protobuf.Timestamp
+//!   - description_with_tags(): &str + {k: v} → ChannelDescriptor
+//!   - EnqueueItem: typed payload enum crossing the sync→async boundary
+//!   - constructors for single/batch points for double/int/string series
+
+use std::collections::HashMap;
+
+use nominal_api::tonic::google::protobuf::Timestamp;
+use nominal_streaming::prelude::*;
+use pyo3::prelude::*;
+use pyo3::types::PyAny;
+use pyo3::types::PyAnyMethods;
+use pyo3::types::PyDateTime;
+use pyo3::types::PySequence;
+
+/// Convert a Python timestamp into google.protobuf.Timestamp.
+///
+/// Accepts:
+/// - int nanoseconds since Unix epoch (UTC)
+/// - datetime.datetime (aware → converted to UTC; naive → treated as UTC)
+///
+/// NOTE: this utilizes the python runtime, and will result in the GIL being held while
+///       running. Future optimizations may involve doing preprocessing in python and
+///       only passing integral timestamps across the language barrier.
+pub fn parse_timestamp(ts_obj: &Bound<'_, PyAny>) -> PyResult<Timestamp> {
```

Comment:
how much work is this to do today? I'd rather just start with a more minimal, int-only interface for this if possible!